app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  title ||= 'Share this page'
+  facebook_href ||= false
+  twitter_href ||= false
+%>
+<% if facebook_href || twitter_href %>
+<div class="app-c-share-links-wrapper">
+  <h2><%= title %></h2>
+
+  <div class="app-c-share-links" data-module="track-share-button-clicks">
```

Comment:
We could probably refactor out this container by:
* moving data module to parent
* applying margin bottom to the title
* applying font size to the links